Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -192,34 +180,56 @@ theorem mem_fixedPoints_iff_card_orbit_eq_one {a : α} [Fintype (orbit M a)] :
 #align mul_action.mem_fixed_points_iff_card_orbit_eq_one MulAction.mem_fixedPoints_iff_card_orbit_eq_one
 #align add_action.mem_fixed_points_iff_card_orbit_eq_zero AddAction.mem_fixedPoints_iff_card_orbit_eq_one
 
-end MulAction
+end FixedPoints
 
-namespace MulAction
+section Stabilizers
 
-variable (G : Type u) {α : Type v} [Group G] [MulAction G α]
+variable {α}
 
-/-- The stabilizer of an element under an action, i.e. what sends the element to itself.
-A subgroup. -/
-@[to_additive
-      "The stabilizer of an element under an action, i.e. what sends the element to itself.
-      An additive subgroup."]
-def stabilizer (a : α) : Subgroup G :=
-  { Stabilizer.submonoid G a with
-    inv_mem' := fun {m} (ha : m • a = a) => show m⁻¹ • a = a by rw [inv_smul_eq_iff, ha] }
-#align mul_action.stabilizer MulAction.stabilizer
-#align add_action.stabilizer AddAction.stabilizer
+/-- The stabilizer of a point `a` as a submonoid of `M`. -/
+@[to_additive "The stabilizer of a point `a` as an additive submonoid of `M`."]
+def stabilizerSubmonoid (a : α) : Submonoid M where
```

Comment:
I'm not totally convinced that this name warrants changing, but I don't have a good alternative. It seems we have no real consistency on how to name different bundlings of the same object (compare: https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/RFC.3A.20names.20for.20different.20bundlings.20of.20the.20same.20function)

Reply:
I changed it because I believed having the submonoid version and the subgroup version in different namespaces was awkward and added a little bit of extra hurdle against finding/discovering one definition with the knowledge of the other. The other option would have been to keep `MulAction.Stabilizer.submonoid` but rename `MulAction.stabilizer` to `MulAction.Stabilizer.subgroup` (and it still is an option). The problem would be exacerbated if I were to add the pointwise stabilizer and the setwise stabilizer, with both monoid and group variants (#6533).